GmailUtils.gs

```diff
@@ -500,8 +500,21 @@ function defaults_(options, defaults) {
  * @return {string}
  */
 function localTimezone_() {
-  var timezone = new Date().toTimeString().match(/\(([a-z0-9]+)\)/i);
-  return timezone.length ? timezone[1] : 'GMT';
+  // 19:26:50 GMT-0700 (Pacific Daylight Time)
+  // 19:29:40 GMT-0700 (PDT)
+  //Logger.log('Hello=' + new Date().toTimeString());
+  //console.log('Hello=' + new Date().toTimeString());
+  //var timezone = new Date().toTimeString().match(/\(([a-z 0-9]+)\)/i);
+  //return timezone.length ? timezone[1] : 'GMT';
+  var tz = gettz_();
+  return tz ? tz : 'GMT';
+}
+
+function gettz_() {
+  var d = new Date(); // now, or the specific date in question
+  var s = d.toLocaleString("en", {timeZoneName: "short"}).split(' ').pop();
```

Comment:
This might  fix part of the issue. In my case (US Mountain), the value is still not supported. The safest way is to get the timezone value from App script itself. I'll send a pull request for the code I have been using.